.github/workflows/ci.yaml

```diff
@@ -25,15 +26,42 @@ jobs:
           path: |
             ~/.cache/pip
             .tox
-          key: ${{ runner.os }}-v1-python-3.8-${{ hashFiles('requirements.txt', 'test-requirements.txt', 'pyproject.toml', 'tox.ini') }}
+          key: ${{ runner.os }}-v1-python-3.8-black-${{ hashFiles('test-requirements.txt', 'pyproject.toml', 'tox.ini') }}
```

Comment:
`test-requirements.txt` -> `requirements.txt`

Reply:
No that's as expected. `back` depends only on the `test-requirements.txt`, - that's where we define it as a test/dev dependency.

Reply:
`pylint` (below) also needs `requirements.txt` as it does more in-depth code analysis for the project, including its pip  dependencies.